source/_integrations/logi_circle.markdown

```diff
@@ -16,6 +16,12 @@ The `logi_circle` implementation allows you to integrate your [Logi Circle](http
 
 ## Requesting API access
 
+<div class='note warning'>
+
+Due to the ongoing situation with COVID-19, Logitech have temporarily stopped processing applications for new API accounts. Existing account should continue to work.
+
+</div>
```

Comment:
Small typo and the additional lines are not needed since there is no language context switching happening inside the message.

```suggestion
<div class='note warning'>
Due to the ongoing situation with COVID-19, Logitech has temporarily stopped processing applications for new API accounts. Existing accounts should continue to work.
</div>
```

Reply:
Yeah I just copied that div from darksky as it was the first place I could think of that had one of those warnings